src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -11,8 +12,24 @@
 [SecurityCritical]
 sealed class DomainManager
 {
+    [StructLayout(LayoutKind.Sequential)]
+    private unsafe struct NativeBootstrapperContext
+    {
+        public char* OperatingSystem;
```

Comment:
I guess so. Runtime ID is supposed to basically be an identifier of the CLR and Native platform being used so I still think it's part of the native stack, but I can see it both ways.